### DISCUSSION TITLE
Welcome to ontology-access-kit Discussions!

### DISCUSSION THREAD

**Discussion by cmungall** [INCATools/ontology-access-kit#246] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samsontu** [comment]
I have been associated with the Stanford University's Protege project for a long time. I am a new user of oak. I am wondering whether this Discussions area is a place for newbies to ask questions on the usage of the tool (like the protege-discussions mailing list for questions related to Protege). Let me know if there is a better forum. 
Thanks.
(lexmatch is giving me "ValueError: ['http'] are used in the SSSOM mapping set but it does not exist in the prefix map" that I can't figure out.)
-Samson Tu

**Reply by matentzn** [reply]
@samsontu this sounds like a real issue rather than a discussion but let's try to troubleshoot. Can you tell me what the exact version of oak is you are using?

**Reply by samsontu** [reply]
@matentzn Let me know if there is a better forum to discuss newbie issues. I don't see a way to get the version from the oak software itself. I installed it in the last couple days using the pip install oaklib command, and it says:
Requirement already satisfied: oaklib in ./venv/lib/python3.9/site-packages (0.1.71). If that's an old version, what is the best way to update it? Thanks.

**Reply by cmungall** [reply]
Hi @samsontu good to see you here!

That's a very old version...

Try this:

`pip install --upgrade oaklib`

Your issue was fixed a while ago:

https://github.com/INCATools/ontology-access-kit/issues/587

**Reply by samsontu** [reply]
Thanks. That's fixed the problem.
